src/modules/CoreUnit/resolvers.ts

```diff
@@ -2,14 +2,15 @@ import { stringArg, list, queryField } from 'nexus/dist';
 
 export const coreUnits = queryField('coreUnits', {
   type: list('CoreUnit'),
-  resolve: (parent, args, ctx) => ctx.prisma.coreUnit.findMany(),
+  resolve: (_parent, _args, ctx) => ctx.prisma.coreUnit.findMany(),
 });
 
 export const coreUnit = queryField('coreUnit', {
   type: 'CoreUnit',
   args: { id: stringArg() },
-  resolve: (parent, { id }, ctx) => {
+  resolve: (_parent, { id }, ctx) => {
     if (!id) {
+      ctx.request.req.log.error('coreUnit resolver: id is undefined');
       throw new Error('please provide id');
     }
     return ctx.prisma.coreUnit.findUnique({
```

Comment:
Let's add logging to the prisma as well. Can you check if an integration available for it like for express? Otherwise:
https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/logging